test/third-party/axios.test.ts

```diff
@@ -125,6 +125,7 @@ it('preserves "auth" (Authorization)', async () => {
 
   // Construct an Axios request with "auth".
   await axios.get(httpServer.http.url('/books'), {
+    adapter: 'http',
```

Comment:
Oh, so the trick was using the HTTP adapter. By default, Axios is using XHR, right? Good catch.